app/views/items/_form.html.erb

```diff
@@ -8,6 +8,12 @@
                  label: false,
                  placeholder: t('items.columns.item_description'),
                  required: false %>
+  <%= form.association :unit_of_measure,
```

Comment:
I assume the association knows to pull in all Unit Of Measures, vs having to pass in a `collection`?

Also, are you able to move the `prompt` into the localization file... if so, this would be cleaner.

Reply:
I went off the below examples.

```
The association helper just invokes input under the hood, so all options available to :select, :radio_buttons and :check_boxes are also available to association. Additionally, you can specify the collection by hand, all together with the prompt:

f.association :company, collection: Company.active.order(:name), prompt: "Choose a Company"
In case you want to declare different labels and values:

f.association :company, label_method: :company_name, value_method: :id, include_blank: false
Please note that the association helper is currently only tested with Active Record. It currently does not work well with Mongoid and depending on the ORM you're using your mileage may vary.
```